{
    "type": "bundle",
    "id": "bundle--d87217cb-33f9-4128-894d-bd27ae4b20c8",
    "spec_version": "2.0",
    "objects": [
        {
            "object_marking_refs": [
                "marking-definition--fa42a846-8d90-4e51-bc29-71d5b4802168"
            ],
            "id": "relationship--21409e96-7a2f-4109-b23e-1b43005aaaa4",
            "type": "relationship",
            "created": "2021-08-18T19:35:07.010Z",
            "created_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5",
            "external_references": [
                {
                    "source_name": "McAfee Maze March 2020",
                    "url": "https://www.mcafee.com/blogs/other-blogs/mcafee-labs/ransomware-maze/",
                    "description": "Mundo, A. (2020, March 26). Ransomware Maze. Retrieved May 18, 2020."
                }
            ],
            "modified": "2021-08-18T19:35:07.010Z",
            "description": "[Maze](https://attack.mitre.org/software/S0449) has checked the language of the machine with function <code>GetUserDefaultUILanguage</code> and terminated execution if the language matches with an entry in the predefined list.(Citation: McAfee Maze March 2020)",
            "relationship_type": "uses",
            "source_ref": "malware--d9f7383c-95ec-4080-bbce-121c9384457b",
            "target_ref": "attack-pattern--c1b68a96-3c48-49ea-a6c0-9b27359f9c19",
            "x_mitre_version": "1.0",
            "x_mitre_modified_by_ref": "identity--c78cb6e5-0c4b-4611-8297-d1b8b55e40b5"
        }
    ]
}